site/sfguides/src/security_analytics_with_wiz_and_snowflake/security_analytics_with_wiz_and_snowflake.md

```diff
@@ -0,0 +1,528 @@
+The following query provides shows the assests with the oldest vulnerabilties age:
+
+```sql
+SELECT
+    assetname, 
+    MAX(DATEDIFF('day', firstdetected, lastdetected)) AS oldest_vuln_age
+FROM WIZDB.WIZSCHEMA.VULNERABILITIES_HISTORICAL
+GROUP BY assetname
+ORDER BY oldest_vuln_age DESC;
+```
+
+## Building some dashboards
+
+Let's use the some on the queries we created in the last sections in order to create a dashbaord where we can track Wiz Issues, vulnerabilities and host configuraiton findings.
+First let's go to the Dashboards page on Snowflake and create a new dashboard. Then let's hit the "+" button and create a "New Tile" and select "From SQL Worksheet".
+Let's place this query we used before for identifying the cloud platform with most Issues:
+```sql
+select 
+        resource_platform, 
+        severity,
+       count(issue_id) 
+from WIZDB.WIZSCHEMA.issues 
+--WIZDB.WIZSCHEMA.issues_historical
+WHERE resource_platform IS NOT NULL AND resource_platform != '' and severity = 'CRITICAL'
+GROUP BY 1,2;
+```
+Now, let's select the Heatgrid as the Chart type.
+We can repeart the same process for other queries we executed against the Issues, vulnerabilties and host configuraiton findings tables and create a dashbaord such as the following:
+![test](assets/p1.png)
+
+<!-- ------------------------ -->
+## Teardown of the Lab
+Duration: 2
+
+Once you have completed the lab, it is important to clean up the resources you have created to avoid incurring unnecessary costs and to keep your environment tidy. Follow these steps to teardown the lab:
+
+### Switch to SYSADMIN Role
+```sql
+USE ROLE SYSADMIN;
+```
+This command sets your current role to SYSADMIN, which has the necessary permissions to manage databases, schemas, and warehouses.
+
+### Drop Tables
+
+```sql
+DROP TABLE IF EXISTS WIZDB.WIZSCHEMA.WIZISSUES;
+DROP TABLE IF EXISTS WIZDB.WIZSCHEMA.WIZVULNERABILITIES;
+DROP TABLE IF EXISTS WIZDB.WIZSCHEMA.hostconfiguration;
+```
+These commands delete the tables WIZISSUES, WIZVULNERABILITIES, and hostconfiguration from the WIZSCHEMA schema in the WIZDB database.
+
+### Drop Warehouse
+```sql
+DROP WAREHOUSE IF EXISTS WIZ_LAB_WH;
+```
+This command deletes the WIZ_LAB_WH warehouse
+
+### Drop Schema
+```sql
+DROP SCHEMA IF EXISTS WIZDB.WIZSCHEMA;
+```
+This command deletes the WIZSCHEMA schema from the WIZDB database
+
+### Drop Database
+```sql
+DROP DATABASE IF EXISTS WIZDB;
+```
+This command deletes the WIZDB database.
+
+### Switch to SECURITYADMIN Role
+
+```sql
+USE ROLE SECURITYADMIN;
+```
+This command deletes the WIZDB database.
+
+### Drop User
+```sql
+DROP USER IF EXISTS CLOUD_SECURITY_ANALYST;
+```
+This command deletes the CLOUD_SECURITY_ANALYST user account.
+
+### Drop Role
+```sql
+DROP ROLE IF EXISTS CLOUD_SECURITY_ANALYST_ROLE;
+```
+This command deletes the CLOUD_SECURITY_ANALYST_ROLE role.
+
+
+
+## Conclusion & Next Steps
```

Comment:
@guy-mast-wiz Change the label to "Conclusion And Resources" and make the format, layout, and content similar to https://quickstarts.snowflake.com/guide/getting_started_with_snowflake_arctic/index.html#5

Reply:
Done